zerver/openapi/markdown_extension.py

```diff
@@ -10,10 +10,12 @@
 import markdown
 
 import zerver.openapi.python_examples
-from zerver.openapi.openapi import get_openapi_fixture, openapi_spec
+from zerver.openapi.openapi import get_openapi_fixture, openapi_spec, get_openapi_description
 
-MACRO_REGEXP = re.compile(r'\{generate_code_example(\(\s*(.+?)\s*\))*\|\s*(.+?)\s*\|\s*(.+?)\s*(\(\s*(.+)\s*\))?\}')
+MACRO_REGEXP = re.compile(r'\{generate_code_example(\(\s*(.+?)\s*\))*\|\s*('
+                          + r'.+?)\s*\|\s*(.+?)\s*(\(\s*(.+)\s*\))?\}')
```

Comment:
What's the story with this change?

Reply:
The linter was showing line too long so I had it to break it down.

Reply:
Hmm, I guess we probably had an exclude rule previously?  I'll investigate a bit.

Reply:
OK I don't see why it passed lint before, but in any case, we should just use a better line-wrapping approach here.